docs/src/extensions.md

```diff
@@ -5,16 +5,22 @@ CurrentModule = JuMP
 Extending JuMP
 ==============
 
+```@meta
 TODO: How to extend JuMP: discussion on different ways to build on top of JuMP.
```

Comment:
`@meta` introduces a code block, so you need a `#` to write text comments inside. This is likely causing the build failures: https://travis-ci.org/JuliaOpt/JuMP.jl/jobs/492287976#L650.

Reply:
Thanks for pointing that out